### PR TITLE
feat: enable reporting to slack 

### PR DESCRIPTION
**Pull request by dewanshparashar**
We'd like to report the unsuccessful retryables on Slack.

This PR derives all the info needed to adhere to our current format and sends them as a message on slack. 

image

Closes FS-525

### CODE REVIEW

**Comment by mahsamoosavi** [comment]
> We'd like to report the unsuccessful retryables on Slack.
> 
> This PR derives all the info needed to adhere to our current format and sends them as a message on slack.
> 
> image
> Closes FS-525

Currently under review

File: lib/report_retryables.ts [hunk 1]
```diff
@@ -0,0 +1,439 @@
...
+  ) {
+    return
+  }
+
+  const l1Report = parentChainTicketReport
+
+  const childChainProvider = new providers.JsonRpcProvider(
+    String(childChain.orbitRpcUrl)
+  )
+  const parentChainProvider = new providers.JsonRpcProvider(
```
Comment (mahsamoosavi) [inline lib/report_retryables.ts:84]: Looks like we don't use the `parentChainProvider` anywhere in the script, should we remove it then?

**Review by mahsamoosavi — CHANGES_REQUESTED** [review]
one single comment, LGTM otherwise.

File: lib/report_retryables.ts (hunk 1 above)
Comment (dewanshparashar) [reply lib/report_retryables.ts]: Updated 👍

**Review by mahsamoosavi — APPROVED** [review]
🚀

**Pull request merged**